src/MovieType.ts

```diff
@@ -0,0 +1,7 @@
+export interface Movie {
```

Comment:
As we learned, and probably it's better to move type file to new directory src/types/MovieType. 
Everything else is great, as for me)